features/attach_validtoken.feature

```diff
@@ -24,7 +24,6 @@ Feature: Command behaviour when attaching a machine to an Ubuntu Advantage
         Then if `<release>` in `focal` and stdout matches regexp:
         """
         \d+ update(s)? can be applied immediately.
-        \d+ of these updates (is a|are) standard security update(s)?.
```

Comment:
@lucasmoura I thought we were adding this message back even when there are 0 updates?

Reply:
No, it is actually the other one we are adding back when we have 0 updates